events/actions.py

```diff
@@ -4,11 +4,14 @@
 Base classes for possible actions taken on events.
 """
 
-from events.conf import settings
-from events.utils import import_object
+from __future__ import unicode_literals
 
+from django.conf import settings
+from events.utils import import_object
 
-_mod_name, _obj_name = settings.EVENTS_STORE_OBJECT.rsplit('.', 1)
+store_object_path = settings.get('EVENTS_STORE_OBJECT',
+                                 "django.core.cache.cache")
```

Comment:
Perhaps we can keep this settings access in events/conf.py? E.g.
```
from django.conf import settings

DEFAULT_STORE_OBJECT = "django.core.cache.cache"
STORE_OBJECT = settings.get('EVENTS_STORE_OBJECT', DEFAULT_STORE_OBJECT)
```
and then here we could do
```
from events import conf
...
_mod_name, _obj_name = conf.STORE_OBJECT.rsplit('.', 1)
```

Reply:
sure thing!